makefiles/platform

```diff
@@ -185,8 +187,89 @@ endif
 		ARFLAGS	+= -xml -o
 
 #	platform specific settings
+ifeq ($(CPU), x86_64)
+
+#		specify flags to cause compiler to generate dependencies
+		MK_DEPS_FLAGS	= -M
+
+#		specify the compiler
+		CC				= gcc
+
+#		specify flags for including files
+		INCLUDE_FLAG	= -I
+		SYSINCLUDE_SEP	=
+		SYSINCLUDE_FLAG	= -isystem
+
+#		specify the library extension
+		LIB_EXTENSION	=
+
+#		specify the optimizer setting
+ifdef RELEASE_BUILD
+#		CPLATFLAGS	+= -O3 -DNDEBUG=1 -- O3 crashes compiler! NDEBUG not in Sequitur
```

Comment:
NDEBUG disables assert() from <assert.h>, if that's used

Reply:
I know it does. That is an old left over comment, I just copied from the x86 block for x86_64 block and forgot to remove. Probably from back from BeOS days, as I don't remember hearing -O3 crashing compiler since.  Removed the comment.